applications/openshift/kubelet/kubelet_configure_tls_cipher_suites_ingresscontroller/rule.yml

```diff
@@ -18,24 +18,36 @@ severity: medium
 #    cce@ocp4:
 
 references:
-  cis@ocp4: 4.2.13
+  cis@ocp4: 4.2.12
 
-ocil_clause: "TLS cipher suite configuration is not configured"
+ocil_clause: "Ingress controller TLS cipher suite configuration is incomplete or possibly insecure"
 
 ocil: |-
```

Comment:
I think the instructions here need to be updated.
It is better to be a description that could help users on how to know why the rule PASS/FAIL.
From the current test result, It will PASS if the policy is custom and the ciphers are in the allowed list:
```
% oc -n openshift-ingress-operator get ingresscontroller/default  -o=jsonpath={.spec.tlsSecurityProfile.custom} | jq -r 
{
  "ciphers": [
    "ECDHE-ECDSA-AES128-GCM-SHA256",
    "ECDHE-ECDSA-CHACHA20-POLY1305",
    "ECDHE-ECDSA-AES256-GCM-SHA384",
    "TLS_CHACHA20_POLY1305_SHA256",
    "TLS_AES_128_GCM_SHA256",
    "TLS_AES_256_GCM_SHA384",
    "ECDHE-RSA-AES128-GCM-SHA256"
  ],
  "minTLSVersion": "VersionTLS12"
}
```

Reply:
Ack.

The `ocil` instruction looks like it's telling users how to fix the issue, which seems redundant with the remediation we ship.

Reply:
Yes, below is the current one. 
```
% oc get ccr upstream-ocp4-cis-kubelet-configure-tls-cipher-suites-ingresscontroller  -o=jsonpath={.instructions}
Run the following command on the kubelet nodes(s):
oc -n openshift-ingress-operator patch ingresscontroller/default --type merge -p '{"spec":{"tlsSecurityProfile":{"type":"Custom","custom":{"ciphers":["ECDHE-ECDSA-AES128-GCM-SHA256","ECDHE-ECDSA-CHACHA20-POLY1305","ECDHE-ECDSA-AES256-GCM-SHA384","TLS_CHACHA20_POLY1305_SHA256","TLS_AES_128_GCM_SHA256","TLS_AES_256_GCM_SHA384","ECDHE-RSA-AES128-GCM-SHA256","ECDHE-RSA-AES256-GCM-SHA384","ECDHE-RSA-CHACHA20-POLY1305"],"minTLSVersion":"VersionTLS12"} } } }'
Is it the case that Ingress controller TLS cipher suite configuration is incomplete or possibly insecure?
```
Better to use `oc get ingresscontrollers/default -n openshift-ingress-operator -o=jsonpath='{.status.tlsProfile.ciphers[:]}'` command to check Ingress controller TLS cipher suite configuration.

Reply:
Thanks for the recommendation - proposed a separate patch to update the instruction.

https://github.com/ComplianceAsCode/content/pull/12355